public/Components/imageButtonComponent.js

```diff
@@ -0,0 +1,37 @@
+class ImageButton extends HTMLElement{
+
+    constructor(){
+
+        super();
+        this.attachShadow({mode:"open"});
+        this.imagePath=this.getAttribute("imagePath") || "";
+        this.buttonId=this.getAttribute("buttonId") || "";
+
+        this.shadowRoot.innerHTML=`
+            <style>
+            #${this.buttonId}{
+                background-image: url("${this.imagePath}");
+                background-repeat: no-repeat;
+                background-size: 100% 100%;
+                background-position: center;
+                border: none;
+                width: 100%;
+                height: 10%;
```

Comment:
```suggestion
                width: 100px;
                height: 100px;
```

public/typhoonQuiz/typhoonQuiz.css の48行でsideAreaの幅が100pxに固定されてるので、そこでしか使わないなら縦横100pxのボタンにしたほうが画面サイズの影響を受けなくていいと思います。